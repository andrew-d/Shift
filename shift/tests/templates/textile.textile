_italics_ and *bold* text!
