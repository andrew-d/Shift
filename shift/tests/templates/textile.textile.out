	<p><em>italics</em> and <strong>bold</strong> text!</p>